blockchain/cache.go

```diff
@@ -48,13 +36,33 @@ type emulatorCache struct {
 	cache *lru.Cache
 }
 
-// reset the cache for the ID.
+// newEmulatorCache returns a new instance of emulatorCache with provided capacity.
+func newEmulatorCache(capacity int) *emulatorCache {
+	cache, err := lru.New(capacity)
+	if err != nil {
+		sentry.CaptureMessage("Continuing without emulator caching: " + err.Error())
+		cache = nil
```

Comment:
not needed. Idiomatic Go always returns nil when err is present

Reply:
also would be better to just do here:
```suggestion
		return &emulatorCache{}
```

Reply:
Removed it